src/sonic-config-engine/minigraph.py

```diff
@@ -221,7 +221,7 @@ def parse_dpg(dpg, hname):
                 intfs_inpc.append(pcmbr_list[i])
                 pc_members[(pcintfname, pcmbr_list[i])] = {'NULL': 'NULL'}
             if pcintf.find(str(QName(ns, "Fallback"))) != None:
-                pcs[pcintfname] = {'members': pcmbr_list, 'fallback': pcintf.find(str(QName(ns, "Fallback"))).text, 'min_links': str(int(math.ceil(len() * 0.75)))}
+                pcs[pcintfname] = {'members': pcmbr_list, 'fallback': pcintf.find(str(QName(ns, "Fallback"))).text}
```

Comment:
I think we should put `pcmbr_list` as len parameter here.

Reply:
I see here decided to remove this parameter https://github.com/Azure/sonic-buildimage/pull/2053#issuecomment-570615588